src/mcs/EmpiricalDistribution.jl

```diff
@@ -1,3 +1,7 @@
+try
+    using ExcelReaders
+end
```

Comment:
There is a way to handle all of the file loading here without taking a dependency on either anything Excel or CSV, we can discuss in our meeting.